### ISSUE TITLE
Updating the layout of the footer section

### ISSUE THREAD

**Issue by Ciggzy1312** [CodeFlow201/codeflow#81]
Would like to replace the links with icons to give it a more cleaner look
Also highlighting the headers of each column and making separate column named "About Us" and "Contact Us"

Would like to work on this issue as an LGMSOC participant

**Comment by ishan267** [comment]
Yes I have understood the objective and would like to work on improving the footer layout as a LGMSOC 21 participant, please assign this issue to me!
